website/docs/configuration/observability.md

```diff
@@ -36,10 +36,38 @@ $ docker logs my-ferretdb
 
 FerretDB writes logs to the standard error (`stderr`) stream.
 
-## Metrics
+## Debug handler
 
-FerretDB exposes metrics in Prometheus format on the debug handler on `http://127.0.0.1:8088/debug/metrics` by default.
-There is no need to use an external exporter.
+FerretDB exposes various HTTP endpoints with the debug handler on `http://127.0.0.1:8088/debug/` by default.
 The host and port can be changed with [`--debug-addr` flag](flags.md#interfaces).
 
-Please note that the set of metrics is not stable yet; metric and label names and formatting of values might change in minor releases.
+### Metrics
+
+FerretDB exposes metrics in Prometheus format on the `/debug/metrics` endpoint.
+There is no need to use an external exporter.
+
+<!-- https://github.com/FerretDB/FerretDB/issues/3420 -->
+
+:::note
+The set of metrics is not stable yet; metric and label names and value formatting might change in minor releases.
+:::
+
+### Probes
+
+FerretDB exposes the following probes that can be used for
+[Kubernetes health checks](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)
+and similar use cases.
+They return HTTP code 200 if a probe is successful and 500 otherwise.
```

Comment:
```suggestion
They return HTTP code 200 if a probe is successful and 5xx otherwise.
```

For the readiness probe it makes more sense to return 503 (at least in some scenarios). The readiness probe is useful, when the service is temporarily unavailable, so it's not killed by kubelet, but it's not flooded with new queries.

Reply:
I updated the docs, but I think we should still return 500, not 503, in practice. Why 503 and not 502, for example? Also, HTTP semantics go beyond status code names. For example, https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/503 says, "a user-friendly page explaining the problem should be sent."